docs/symfony-messenger.md

```diff
@@ -0,0 +1,72 @@
+# Using async Symfony Messenger
+
+Add new service to the `compose.yaml`:
+```yaml
+  php-worker:
+    image: ${IMAGES_PREFIX:-}app-php-worker
+    restart: unless-stopped
+    environment:
+      - RUN_MIGRATIONS=false
+    healthcheck:
+      disable: true
+    depends_on:
+      - php
+```
+
+Add new services to the `compose.override.yaml`:
+```yaml
+  php-worker:
+    profiles:
+      - donotstart
```

Comment:
could you explain what's this for? :blush:

Reply:
Hello @norkunas, in `compose.yaml` we have **base** `php-worker` which is extended by `php-worker-async` in `compose.override.yaml` and in `compose.prod.yaml`, `donotstart` is used to prevent separate `php-worker` starting which is only used as a base.

What do you think, is it necessary to add explanation for that in `docs/symfony-messenger.md`?

Reply:
Understood, thanks for explanation :+1: 
I think it could be good to explain for others like me which sees this first time :)

Reply:
Just another question: why not naming these services same? IMHO worker is already basically async because it's another service

Reply:
> Just another question: why not naming these services same? IMHO worker is already basically async because it's another service

By using different name it is easy to add additional workers with different configuration. I updated `symony-messenger.md` to try clarify that.